amp/WEB-INF/src/org/digijava/module/aim/annotations/activityversioning/ActivityVersioningService.java

```diff
@@ -0,0 +1,366 @@
+package org.digijava.module.aim.annotations.activityversioning;
```

Comment:
since we already have a util class for activity versioning we could have used that class, as I did. we also have methods related to comparison in that class. then we would have to create a ticket as followup to provide a proper separation of classes, based on functionality